packages/react-devtools-shared/src/devtools/views/Settings/SettingsContext.js

```diff
@@ -609,6 +277,11 @@ export function updateThemeVariables(
     // $FlowFixMe scrollbarColor is missing in CSSStyleDeclaration
     documentElement.style.scrollbarColor = `var(${`--${theme}-color-scroll-thumb`}) var(${`--${theme}-color-scroll-track`})`;
   });
+
+  // @reach/tooltip create the tooltip element outside the ThemeProvider,
+  // so set the tooltip styles in the root when the display preference changes.
```

Comment:
Nice catch 😄 

This comment might be a little confusing though, since the React tooltips _are_ within the "provider" just not within its DOM subtree. Maybe we could change the wording slightly?

```suggestion
  // The ThemeProvider works by writing DOM style variables to an HTMLDivElement.
  // Because Portals render in a different DOM subtree, these variables don't propagate.
  // So we need to also set @reach/tooltip specific styles on the root.
```

Reply:
Sure, that's much more accurate